### PR TITLE
Adding proper error message for Camera config page

### LINKED ISSUE #7620: Add proper error message for Camera configuration page
**Describe the bug**

When we enter the wrong middleware for a camera configuration, it throws "something went wrong" error

**To Reproduce**

1. Go to camera asset configuration page
2. input wrong middleware and submit
3. See error

**Expected behavior**

It should be throwing a proper error

**Screenshots**

image

### PR DESCRIPTION
**Pull request by hrit2773**
## Proposed Changes

- Fixes #7620

### CODE REVIEW

**Comment by hrit2773** [comment]
@nihal467 updated the error message to wrong configuration credentials. can you plz review this

**Comment by hrit2773** [comment]
@nihal467 can i get a review on this PR is the cypress test failing due to timeout issues

**Comment by nihal467** [comment]
LGTM

File: src/Redux/fireRequest.tsx [hunk 1]
```diff
@@ -158,9 +158,15 @@ export const fireRequest = (
...
               });
             } else {
-              Notification.Error({
-                msg: "Something went wrong...!",
-              });
+              if (key === "operateAsset") {
+                Notification.Error({
+                  msg: "Wrong Middleware Hostname...!",
+                });
+              } else {
```
Comment (khavinshankar) [inline src/Redux/fireRequest.tsx:165]: This seems like a hack. Also, I thought we moved to useQuery. Please check `src/Utils/request/useQuery.ts`. If the middleware API calls still use fireRequest, please consider refactoring it to useQuery.

cc: @rithviknishad

**Review by khavinshankar — CHANGES_REQUESTED** [review]

File: src/Redux/fireRequest.tsx (hunk 1 above)
Comment (hrit2773) [reply src/Redux/fireRequest.tsx:165]: @khavinshankar the middleware api call uses fire request.. of course I can change it to useQuery if @rithviknishad agrees to it

Comment (rithviknishad) [reply src/Redux/fireRequest.tsx:165]: The existing usages of `operateAsset` was hard to switch to using useQuery/request as there were places where we were passing the redux's dispatch as params to the functions inside it.

However, switching to newer camera feed components would solve the issue as it is not using the deprecated `operateAsset`. (Eg: https://github.com/coronasafe/care_fe/pull/7654)

The newer camera feed components handles the error messages as expected and wouldn't have this underlying issue in the first place.

@hrit2773 Do you want to switch to using the newer camera feed components in the asset config page? If yes, you can make a separate PR.

**Comment by nihal467** [comment]
@hrit2773 any update on the PR

**Comment by bodhish** [comment]
Closing as we haven't moved ahead.

**Pull request closed without merging**